src/translator_testing_model/schema/translator_testing_model.yaml

```diff
@@ -243,7 +244,12 @@ classes:
           be defined as filters to specify TestAsset membership in a "Block List" collection.
   
   PathfinderTestCase:
```

Comment:
Should this not model closely to TestCase? If TestCase includes input category and id, then I think this should as well, split into source and target as it is in the test asset.

Reply:
I can add it back in, but from the Test Harness perspective, these are unused fields and redundant in the Test Assets. I'm inclined to leave them out. What are your thoughts?